<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-REEGCHG" name="Sample_Test4" tests="1" failures="0" timestamp="7 Sep 2016 18:16:54 GMT" time="0.921" errors="0">
  <testcase name="@AfterClass destroy" time="0.0" classname="org.mobile.tests.demoTestCase2">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at org.mobile.tests.demoTestCase2.destroy(demoTestCase2.java:220)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- @AfterClass destroy -->
  <testcase name="sampleTestCase" time="0.002" classname="org.mobile.tests.demoTestCase2"/>
</testsuite> <!-- Sample_Test4 -->
